internal/operands/common-instancetypes/reconcile_test.go

```diff
@@ -165,6 +171,93 @@ var _ = Describe("Common-Instancetypes operand", func() {
 		Expect(cleanupResult).To(BeNil())
 		Expect(err).ToNot(HaveOccurred())
 	})
+
+	It("should create and cleanup common-instancetypes resources from an external URL", func() {
+		request.Instance.Spec.CommonInstancetypes = &ssp.CommonInstancetypes{
+			URL: pointer.String("https://github.com/kubevirt/common-instancetypes"),
```

Comment:
In my opinion it's unusual that unit tests depend on the contents of an external URL.
But in this case, I'm not sure how it can be avoided.

Reply:
Fully agree, the only thing I can do here is create a nested kustomize target within the tree and use `file://` as we aren't going through the webhooks here. This doesn't exercise as much code but works offline.